.github/workflows/e2e-test.yml

```diff
@@ -1,4 +1,4 @@
-name: E2E Testing w/ Playwright
```

Comment:
Nice. I never liked this name. I named it this, because I was copying another repo.